### PR TITLE
Components

### PR DESCRIPTION
**Pull request by luqmanmusah**
Created a directory called components.
Inside components, I created a new Calculator.js file.
In Calculator.js, created a React component that matches this design UI.
Once the Calculator was ready, I imported it and use it in my main component (App.js).
Test it in the browser; it renders without problems.
I used class components.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+    render = () => (
+      <div className="grid container max-width-sm">
+        <p value={0} className="result">0</p>
```
Comment (vichuge) [inline src/components/Calculator.js:13]: - [OPTIONAL] Following the [given design](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A), you could improve the display number on the calculator, why not make a color white text, better center, and bigger? That'd improve your project.
image
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] At this point, there are unnecessary files that could be deleted to have a clean project. Please consider deleting all these elements.
image


- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luqmanmusah** [comment]
Fixed all the comments mentioned above.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**